client/src/modules/stock/entry/entry.js

```diff
@@ -431,6 +431,15 @@ function StockEntryController(
         .then((lots) => {
           item.availableLots = lots;
           item.candidateLots = lots.filter(lot => !lot.expired);
+
+          return vm.depot && vm.depot.uuid ? Stock.inventories.read(null, {
```

Comment:
Could you just get the single inventory item already retrieved on line 677?
> const inventory = inventoryStore.get(line.inventory_uuid);

Reply:
The `inventoryStore.get(line.inventory_uuid);` function allows only to get inventory metadata it means without consideration of stock information (quantity in depot or unit cost or wac); in our case we need to know stock information about that given inventory.

Reply:
ok